docs/jobs/ai-assistant.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Using OpenFn with AI Assistant
```

Comment:
We should call it "The AI Assistant" across our docs, otherwise it reads as bad English